doc/user/content/manage/dbt/_index.md

```diff
@@ -588,6 +635,25 @@ Component                            | Value     | Description
     indexes=[{'default': True}]) }}
 ```
 
+### Configuration: refresh strategies {#configuration-refresh-strategies}
+
+{{< private-preview />}}
+
+**Minimum requirements:** `dbt-materialize` v1.7.3+
+
+Use the `refresh_interval` configuration to define [refresh strategies](#using-refresh-strategies)
+for materialized view models.
+
+The `refresh_interval` configuration can have the following components:
+
+Component       | Value    | Description
+----------------|----------|--------------------------------------------------
+`at`            | `string` | The specific time to refresh the materialized view at, using the [refresh at](/sql/create-materialized-view/#refresh-at) strategy.
```

Comment:
Maybe "The specific time at which to refresh the materialized view. Uses the [refresh at](... strategy)"?

For on_commit -- we use "configures the ..." ... Should this also be configures?

Reply:
Both `on_commit` and `at_creation` are booleans, while `at` and `every` are strings. This comment made me realize that I didn't document the booleans in a consistent way to the rest of the documentation, so I'll adjust.